src/debugger/handler_dbgp.c

```diff
@@ -1028,7 +1028,7 @@ static int xdebug_do_eval(char *eval_string, zval *ret_zval)
 	zend_execute_data *original_execute_data = EG(current_execute_data);
 	int                original_no_extensions = EG(no_extensions);
 	zend_object       *original_exception = EG(exception);
-	jmp_buf           *original_bailout = EG(bailout);
+	JMP_BUF           *original_bailout = EG(bailout);
```

Comment:
Can you explain what `jmp_buf` vs `JMP_BUF` does?  It's hard too google for :-)

Reply:
It s all about portability, on Linux it is correct however on FreeBSD for instance it is not, thus using the "typedef" version instead.

Reply:
Do you have some reading material about this? I'd love to read up on this.

Reply:
Into Zend portability when defining this type.